pgkit/application/models/replica.py

```diff
@@ -257,7 +257,8 @@ def _get_current_delay(self):
         config = read_file(file_location)
         for line in config.split('\n'):
             if 'recovery_min_apply_delay' in line:
```

Comment:
This doesn't fix commented `recovery_min_apply_delay` case.

Reply:
We now use `line.startswith` in the if condition. This resolves the commented case.